xml/admin_monitoring_alerting.xml

```diff
@@ -35,1024 +37,346 @@
   <listitem>
    <para>
     The <systemitem class="daemon">prometheus-node_exporter</systemitem>
```

Comment:
`prometheus-node_exporter` is very near on the name of the package, but doesn't fit compared to the other entries. I'd suggest to leave out the Prometheus in front of NE and write `Node exporter` instead. If it is preferable to say "Prometheus Alertmanager" and "Prometheus Node exporter" that's probably also fine.